crawler/helpers.py

```diff
@@ -97,41 +109,65 @@ def check_for_required_fields(csvreader: DictReader, centre: Dict[str, str]) ->
     Returns:
         List[str] -- list of errors experienced
     """
-    logger.debug("Checking data for required fields")
-
-    if csvreader.fieldnames and centre["barcode_field"] not in csvreader.fieldnames:
-        error = "Barcode field not in CSV file"
-        logger.error(error)
-        return [error]
+    """Checks that the CSV file has the required headers.
```

Comment:
Might be a pythonism I'm missing, but is there a reason this method has two docstrings? Is it a way of indicating that the method will either Return OR Raise?

Reply:
Actually, is the former docstring still accurate? We don't seem to return a list of errors any more.

Reply:
Good spot, I included an old docstring while updating the method.